.web-docs/metadata.hcl

```diff
@@ -22,13 +22,15 @@ integration {
   ]
   docs {
     process_docs = true
-    # We recommend using the default readme_location of just `./README.md` here
-    # This projects README needs to document the interface of an integration.
-    #
-    # If you need a separate README from what you will display on GitHub vs
-    # what is shown on HashiCorp Developer, this is totally valid, though!
-    readme_location = "./INTEGRATION_README.md"
-    external_url = "https://github.com/hashicorp/integration-template"
```

Comment:
Given we talk about `external_url` below, do we want to remove this reference?

Though is this `external_url` still supported from a plugin developer's perspective? My understanding was that this belonged to the `integrations` repo where we register plugins for them to appear in our docs

Reply:
Great questions... This `external_url` value seems to be supported here in `metadata.hcl` config ([ref](https://github.com/hashicorp/integration-template?tab=readme-ov-file#metadata-configuration)).

I think in 99% of cases it'll be the same as the `repo_url` set up in config submitted to the `integrations` repo, but I believe the _intent_ here was to allow some flexibility in what URL is ultimately linked to from the `developer.hashicorp.com` page, ie this link:

![CleanShot 2024-05-27 at 18 10 46@2x](https://github.com/hashicorp/packer-plugin-scaffolding/assets/4624598/d13fe4fe-fa5b-45d6-a4dd-28ea39fc688c)

However, looking at `developer.` code [we do seem to use `repo_url` for the link above](https://github.com/hashicorp/dev-portal/blob/9a396e6119f261e016602ac9ad41a64646075b26/src/layouts/product-integration-layout/index.tsx#L189), but with a Packer-specific conditional 🤔  I think ideally we'd use this configured URL instead... so my initial feeling is that we should continue to encourage plugin authors to provide it, and update our front-end to accurately reflect it.